content/apt/apt-3/contents.lr

```diff
@@ -27,9 +27,9 @@ Then replace the address in the lines added before with, for example:
    deb tor://sdscoq7snqtznauu.onion/torproject.org tor-nightly-master-<DISTRIBUTION> main
    ```
 
-Replace `<DISTRIBUTION>` with your Operating System codename. Run `lsb_release -c` or `cat /etc/debian_version` to discover.
+Replace `<DISTRIBUTION>` with your Operating System codename. Run `lsb_release -c` or `cat /etc/debian_version` to check the Debian version you're running.
```

Comment:
Replace "check the Debian version" to "check the Operating System version", since it's possible to use our APT repository in Ubuntu and OS based on Debian.